client/src/components/AuthHeader/AuthHeader.tsx

```diff
@@ -3,24 +3,37 @@ import Box from '@material-ui/core/Box';
 import { Link } from 'react-router-dom';
 import useStyles from './useStyles';
 import { Typography } from '@material-ui/core';
+import Logo from '../../Images/logo.png';
+import AuthMenu from '../AuthMenu/AuthMenu';
 
 interface Props {
-  linkTo: string;
+  linkTo?: string;
   asideText: string;
-  btnText: string;
+  btnText?: string;
 }
 
-const AuthHeader = ({ linkTo, asideText, btnText }: Props): JSX.Element => {
+const AuthHeader = ({ asideText }: Props): JSX.Element => {
```

Comment:
Although it wasn't a part of the ticket, good work editing the header to match the mocks 😄